betelgeuse.py

```diff
@@ -177,6 +177,10 @@ def test_case(path, collect_only, project):
                 test.docstring = test.docstring.encode(
                     'ascii', 'xmlcharrefreplace')
 
+            # Is the test automated? Acceptable values are:
+            # automated, manualonly, and notautomated
+            auto_status = 'notautomated' if test.automated else 'automated'
```

Comment:
Since `test.automated` is reported by ``Testimony``, we either get a `True` or `False`.